Apromore-Custom-Plugins/Calendar-Portal/src/main/java/org/apromore/plugin/portal/calendar/controllers/DeleteConfirm.java

```diff
@@ -48,7 +48,7 @@
 @VariableResolver(org.zkoss.zkplus.spring.DelegatingVariableResolver.class)
 public class DeleteConfirm extends SelectorComposer<Window> implements LabelSupplier {
 
-    private CalendarModel calendarItem;
+    private org.apromore.plugin.portal.calendar.model.Calendar calendarItem;
```

Comment:
We can use Calendar here instead of org.apromore.plugin.portal.calendar.model.Calendar